fe/fe-core/src/main/java/org/apache/doris/common/util/DynamicPartitionUtil.java

```diff
@@ -240,6 +240,7 @@ public static void registerOrRemoveDynamicPartitionTable(long dbId, OlapTable ol
         if (olapTable.getTableProperty() != null
                 && olapTable.getTableProperty().getDynamicPartitionProperty() != null) {
             if (olapTable.getTableProperty().getDynamicPartitionProperty().getEnable()) {
+                Catalog.getCurrentCatalog().getDynamicPartitionScheduler().executeDynamicPartitionFirstTime(dbId, olapTable.getId());
```

Comment:
`registerOrRemoveDynamicPartitionTable()` is also called in `replayCreateTable()`.
And in "replay" process, we should not call `executeDynamicPartition()`